src/fontstash.h

```diff
@@ -781,6 +822,12 @@ void fonsSetFont(FONScontext* stash, int font)
 	fons__getState(stash)->font = font;
 }
 
+void fonsSetOversample(FONScontext* stash, int oversampleH, int oversampleV)
+{
+	fons__getState(stash)->oversampleH = oversampleH;
```

Comment:
I think a toggle would be just fine as per #266. Oversample amount could be just a define. I don't think FT subpixel rendering will work directly here.

What I'm expect the stb_tt to do is to render the glyph twice the size, and box blur it. Then when rendering we render the glyph half the size. This effectively gives us "more samples" when rendering. If I understand correctly, FT sub pixel rendering just renders the glyph, but does not snap to pixels. So we should do double size rendering and blur on FT too.